eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py

```diff
@@ -70,14 +71,12 @@ def create_table(private_config_xml_file):
 
 
 def fill_in_table_from_remapping(private_config_xml_file, release_version, reference_directory):
-    query_retrieve_info = (
-        "select taxonomy, scientific_name, assembly_accession, string_agg(distinct source, ', '), sum(num_ss_ids)"
-        "from eva_progress_tracker.remapping_tracker "
-        f"where release_version={release_version} "
-        "group by taxonomy, scientific_name, assembly_accession")
+    query_assembly_accession_info = (f"""select taxonomy, scientific_name, assembly_accession, string_agg(distinct source, ', '), 
+    sum(num_ss_ids) from eva_progress_tracker.remapping_tracker where release_version={release_version} 
+        group by taxonomy, scientific_name, assembly_accession""")
```

Comment:
Totally optional but I personally find the original formatting of line breaks more readable...

Reply:
reverted